packages/volto-slate/locales/it/LC_MESSAGES/volto.po

```diff
@@ -0,0 +1,148 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language: \n"
+"Language-Team: \n"
+"Content-Type: \n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#: editor/plugins/Link/index
+msgid "Add link"
+msgstr "Aggiungi un link"
+
+#: widgets/HtmlSlateWidget
+msgid "An error has occurred while editing "{name}" field. We have been notified and we are looking into it. Please save your work and retry. If the issue persists please contact the site administrator."
+msgstr "E' occorso un errore durante la modifica del campo "{name}". Siamo stati notificati e stiamo indagando. Si prega di salvare il lavoro e riprovare. Se il problema persiste, contattare l'amministratore del sito."
```

Comment:
I'm not a po files expert, but this syntax without escaping the `"` looks weird to me. Does it really work?

Reply:
I think that they go scaped indeed `\"`.